hipercam/scripts/fits2hcm.py

```diff
@@ -499,6 +505,44 @@ def fits2hcm(args=None):
                     ohdul = fits.HDUList([ophdu, ofhdu])
                     ohdul.writeto(oname, overwrite=overwrite)
 
+                elif origin == "OSIRIS":
+                    # Copy main header into primary data-less HDU
+                    ihead = hdul[0].header
+                    ophdu = fits.PrimaryHDU(header=ihead)
+                    ophdu.header["NUMCCD"] = (1, "CCD number; fits2hcm")
+                    exptime = ihead["EXPTIME"]
+                    mjd = ihead["MJD-OBS"] + exptime / 2 / 86400
+                    time = Time(mjd + exptime / 2 / 86400, format="mjd")
+                    ophdu.header["TIMSTAMP"] = (time.isot, "Time stamp; fits2hcm")
+
+                    ofhdu = fits.ImageHDU(hdul[0].data)
+
+                    ofhdu.header["CCD"] = ("1", "CCD label")
+                    ofhdu.header["NXTOT"] = (4096, "Total unbinned X dimension")
+                    ofhdu.header["NYTOT"] = (4112, "Total unbinned Y dimension")
+                    ofhdu.header["NUMWIN"] = (ihead['NUM_ROIS'], "Total number of windows")
+                    ofhdu.header["WINDOW"] = ("1", "Window label")
+                    ofhdu.header["LLX"] = (1, "X-ordinate of lower-left pixel")

```

Comment:
It would be good to get our hands on some windowed data to add this, but OK for now